Gallery.Api/Controllers/CardController.cs

```diff
@@ -40,6 +41,10 @@ public CardController(ICardService cardService, IAuthorizationService authorizat
         [SwaggerOperation(OperationId = "getCards")]
         public async Task<IActionResult> Get(CancellationToken ct)
         {
+            if (!await _authorizationService.AuthorizeAsync([SystemPermission.ViewCollections], ct) ||

```

Comment:
this can be combined into one `await _authorizationService.AuthorizeAsync([SystemPermission.ViewCollections, SystemPermission.ViewExhibits], ct)` I think